[Bug] TypeScript compilation error: createActionSchema not found

## Problem

TypeScript compilation fails with error:
```
src/lib/validation.ts(67,53): error TS2552: Cannot find name 'createActionSchema'. Did you mean 'authActionSchema'?
```

The `CreateContentFormInput` type references `createActionSchema` which does not exist. It should reference `createContentFormSchema` instead.

## Location

`src/lib/validation.ts:67`

## Suggested Fix

Change line 67 from:
```typescript
export type CreateContentFormInput = z.infer<typeof createActionSchema>;
```
to:
```typescript
export type CreateContentFormInput = z.infer<typeof createContentFormSchema>;
```

---
_Auto-created by overnight-dev-agent_

--- a/src/lib/validation.ts
+++ b/src/lib/validation.ts
@@ -64,7 +64,7 @@ export const createContentFormSchema = z.object({
 });
 
 export type CreateContentJsonInput = z.infer<typeof createContentJsonSchema>;
-export type CreateContentFormInput = z.infer<typeof createActionSchema>;
+export type CreateContentFormInput = z.infer<typeof createContentFormSchema>;
 
 // Output update schema
 export const updateOutputSchema = z.object({

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -1,7 +1,9 @@
 {
   "compilerOptions": {
     "target": "ES2020",
-    "lib": ["ES2020"],
+    "lib": [
+      "ES2020"
+    ],
     "jsx": "preserve",
     "strict": true,
     "esModuleInterop": true,
@@ -13,14 +15,26 @@
     "forceConsistentCasingInFileNames": true,
     "incremental": true,
     "paths": {
-      "@/*": ["./src/*"]
+      "@/*": [
+        "./src/*"
+      ]
     },
     "plugins": [
       {
         "name": "next"
       }
-    ]
+    ],
+    "allowJs": true,
+    "noEmit": true,
+    "module": "esnext"
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
-  "exclude": ["node_modules"]
+  "include": [
+    "next-env.d.ts",
+    "**/*.ts",
+    "**/*.tsx",
+    ".next/types/**/*.ts"
+  ],
+  "exclude": [
+    "node_modules"
+  ]
 }

